nemo/core/classes/modelPT.py

```diff
@@ -240,6 +245,75 @@ def register_artifact(
 
         return self._save_restore_connector.register_artifact(self, config_path, src, verify_src_exists)
 
+    def has_artifacts(self) -> bool:
+        """Returns True if model has artifacts registered"""
+        return hasattr(self, 'artifacts') and self.artifacts is not None and len(self.artifacts) > 0
+
+    def has_native_or_submodules_artifacts(self) -> bool:
```

Comment:
Im wondering if these functions should be inside of SaveRestoreConnector and just forward the calls here. Im ok with this also - @ericharper